standalone_self_upgrade.cf.in

```diff
@@ -541,6 +542,10 @@ bundle agent cfengine_master_software_content
       "dir[centos_5_x86_64]" string => "$(dir[redhat_5_x86_64])";
       "dir[oracle_5_x86_64]" string => "$(dir[redhat_5_x86_64])";
       "dir[SuSE_11_x86_64]" string => "$(dir[redhat_5_x86_64])";
+      "pkg[SuSE_12_x86_64]" string => "$(pkg[redhat_6_x86_64])";
```

Comment:
update the comment above here? 

``` # Redhat/Centos/Oracle 5 and SuSE 11 all use the same package```

Should be same as other comment right? Maybe tighten that earlier comment up against the block of vars lines also?

``` # Redhat/Centos/Oracle 6, SLES 12-15 use the same package```

Reply:
right